pkgs/top-level/all-packages.nix

```diff
@@ -16942,7 +16942,7 @@ with pkgs;
     inherit (darwin.apple_sdk.frameworks) ApplicationServices;
   };
 
-  pangomm_2_48 = callPackage ../development/libraries/pangomm/2.48.nix {
+  pangomm_2_50 = callPackage ../development/libraries/pangomm/2.50.nix {
```

Comment:
This should still be 2_48 since the package name refers to ABI version: https://gitlab.gnome.org/GNOME/pangomm/-/blob/cd00ba55a53b71a41fb566827db8b7d7c28c36b6/meson.build#L13